crates/evm/traces/src/lib.rs

```diff
@@ -34,7 +39,88 @@ pub use decoder::{CallTraceDecoder, CallTraceDecoderBuilder};
 pub mod debug;
 pub use debug::DebugTraceIdentifier;
 
-pub type Traces = Vec<(TraceKind, CallTraceArena)>;
+pub type Traces = Vec<(TraceKind, SparsedTraceArena)>;
+
+/// Trace arena keeping track of ignored trace items.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SparsedTraceArena {
+    #[serde(flatten)]
+    pub arena: CallTraceArena,
+    #[serde(default, skip_serializing_if = "HashMap::is_empty")]
+    pub ignored: HashMap<(usize, usize), (usize, usize)>,
```

Comment:
not immediately obvious here what these tuples are.

could we also add some docs from the `IgnoredTraces` here?